skglm/estimators.py

```diff
@@ -117,53 +250,53 @@ def __repr__(self):
             % (self.datafit.__class__.__name__, self.penalty.__class__.__name__,
                self.penalty.alpha, self.is_classif))
 
-    def path(self, X, y, alphas, coef_init=None, return_n_iter=False, **params):
-        """Compute regularization path.
+    # def path(self, X, y, alphas, coef_init=None, return_n_iter=False, **params):
```

Comment:
We have two big chunks of commented code in ``GeneralizedLinearEstimator``, What are we planning to do with them? (remove, keep for future refs, )

Reply:
they can be removed

Reply:
To my opinion, the code in fit can be removed as it can now be found almost completely in `_glm_fit`. I don't really see reasons to keep it but maybe the others will think otherwise.